tests/suites/test_suite_cipher.arc4.data

```diff
@@ -90,15 +90,15 @@ ARC4 Encrypt and decrypt 16 bytes in multiple parts 4
 depends_on:MBEDTLS_ARC4_C
 enc_dec_buf_multipart:MBEDTLS_CIPHER_ARC4_128:128:15:1:-1:15:1:15:1
 
-ARC4 Encrypt and decrypt 22 bytes in multiple parts 1
+ARC4 Encrypt and decrypt 22 bytes in multiple parts 1 [#1]
 depends_on:MBEDTLS_ARC4_C
 enc_dec_buf_multipart:MBEDTLS_CIPHER_ARC4_128:128:15:7:-1:15:7:15:7
 
-ARC4 Encrypt and decrypt 22 bytes in multiple parts 1
+ARC4 Encrypt and decrypt 22 bytes in multiple parts 1 [#2]
```

Comment:
Please remove the number at the end of these test three test cases.

Reply:
No, otherwise the descriptions would not be unique.

Reply:
What I mean is - there are a couple of test cases that end with `parts 1 [#n]` in succesion. It should be either parts 1, parts 2, etc. , or parts [#1], parts [#2].